### ISSUE TITLE
new changes to stream field content is lost if a validation error occurred while saving

### ISSUE THREAD

**Issue by asankamal** [noripyt/wagtail-react-streamfield#29]
I've created a custom ImageGalleryBlock and ImageGalleryItemBlock as follow.

```
class ImageGalleryBlock(StructBlock):
    image_list = ListBlock(ImageGalleryItemBlock(), label="Image Gallery")
    title = CharBlock(label="Gallery Title")

class ImageGalleryItemBlock(StructBlock):
    image = ImageChooserBlock()
    caption = CharBlock(required=False)
```

When I try to save and there is validation error in the one of these blocks, react-streamfield crashes.(e.g: trying to save without 'title' for ImageGalleryBlock)
It shows `Uncaught TypeError: Cannot read property 'split' of null` in console.

**Comment by BertrandBordage** [comment]
I can confirm it, introduced recently by noripyt/react-streamfield@d76a986a536adf941563ac5e477c8f549c43730e & noripyt/react-streamfield@0ee27e093c4404d3a8058d6fcf911caa20aee886. I’m working on a fix.

**Comment by BertrandBordage** [comment]
Fixed by https://github.com/noripyt/react-streamfield/commit/9adafc927d81aa758273e27b831dcfd42af62f0c and released in 1.1.1.
